modules/ROOT/pages/6.0-release-notes-apis.adoc

```diff
@@ -0,0 +1,209 @@
+. The `enable()` and `disable()` functions are now the single `setEnabled(<state>)` function.
+. The `disable(<name>)` and `enable(<name>)` functions are now the single `setEnabled(<name>, <state>)` function in the Dialog APIs.
+
+As well, the `.disabled` property is now the `.enabled` property in all {productname} UI classes.
+
+[[dom-domutils]]
+==== `dom.DOMUtils`
+
+The `toHex` function has been removed from `dom.DOMUtils`.
+
+[[domparser]]
+==== `DomParser`
+
+The `filterNode` method has been removed from the `DomParser` API.
+
+As well, the `DomParser` API no longer uses a custom parser internally and instead uses the https://developer.mozilla.org/en-US/docs/Web/API/DOMParser[native `DOMParser` API].
+
+[[editor-setcontent]]
+==== `editor.SetContent`
+
+The `SetContent` editor event’s `content` property has been deprecated.
+
+[[env]]
+==== `Env`
+
+The following methods have been removed from the `Env` API:
+
+* `opera`
+* `webKit`
+* `ie`
+* `gecko`
+* `mac`
+* `iOS`
+* `android`
+* `desktop`
+* `contentEditable`
+* `caretAfter`
+* `range`
+* `ceFalse`
+* `experimentalShadowDom`
+* `fileApi`
+
+Note, in particular, the removal of all legacy browser detection methods.
+
+[[fire]]
+==== `fire` functions renamed in all {productname} APIs
+
+The `fire()` function in `tinymce.Editor`, `tinymce.dom.EventUtils`, `tinymce.dom.DOMUtils`, `tinymce.util.Observable` and `tinymce.util.EventDispatcher` has been renamed to `dispatch()`.
+
+`fire` has been aliased to `dispatch` but has also been marked as _deprecated_.
+
+[[fixed-apis]]
+=== Fixed APIs
+
+[[editor-annotator-remove]]
+==== `editor.annotator.remove`
+
+Previously, `editor.annotator.remove` scrolled to the removed material’s position when removing an annotation.
+
+It did not, as expected, retain the current selection or insertion point position in the {productname} editor.
+
+With this release, `editor.annotator.remove` does not change the change the selection or insertion point position after removing an annotation.
+
+[[editor-fire]]
+==== `editor.fire()`
+
+Previously, if the {productname} editor was removed, the `editor.fire()` API returned an incorrect object.
+
+In {productname} 6.0, `editor.fire()` returns the correct object even when the editor is removed.
+
+[[editor-getcontent]]
+==== `editor.getContent()`
+
+The `editor.getContent()` API can now provide custom content by preventing and overriding `content` in the `BeforeGetContent` event.
+
+This makes it consistent with the `editor.selection.getContent()` API.
+
+The `editor.setContent()` API can also now be prevented from using the `BeforeSetContent` event.
+
+This, too, makes it consistent with the `editor.selection.setContent()` API.
+
+[[editor-selection-getcontent]]
+==== `editor.selection.getContent()`
+
+The `no_events` argument is Boolean. If set to `true`, events should not fire when `getContent` is called. Previously, `editor.selection.getContent()` did not respect this.
```

Comment:
This line needs rewording. Maybe something along this line
```Previously, `editor.selection.getContent()` did not respect `no_events` argument, which if set to `true`, events should not fire when `getContent` is called```

Reply:
```suggestion
When the `no_events` argument is set to `true`, events should not fire when `getContent` is called. Previously, `editor.selection.getContent()` did not respect this.
```
I think we need to keep the point order for technical accuracy, but that should be better?